### ISSUE TITLE
remote-ls error: [7] Couldn't connect to server

### ISSUE THREAD

**Issue by mcatanzaro** [flatpak/flatpak#1439]
This probably should never happen:

```$ flatpak remote-ls
error: [7] Couldn't connect to server
```

**Comment by mwleeds** [comment]
If you run it with `G_MESSAGES_DEBUG=all` you should be able to tell which remote summary was being fetched when the error happened

**Comment by mcatanzaro** [comment]
I reported #1446 for that.

**Comment by mwleeds** [comment]
I think if it showed the results from the remotes that are up and ignored the one that's down that might be a bit confusing for the user

**Comment by mwleeds** [comment]
But it could be reasonable to do both: show the results that are available and show an error message for the ones that aren't

**Comment by mcatanzaro** [comment]
> But it could be reasonable to do both: show the results that are available and show an error message for the ones that aren't

Yes, it shouldn't silently drop results, but it should show something.

**Comment by hfiguiere** [comment]
`flaptak remote-ls -v` show more details.
